apps/site/assets/package.json

```diff
@@ -102,7 +102,7 @@
     "mocha-jsdom": "2.0.0",
     "postcss-loader": "^6.2.1",
     "postcss-preset-env": "^7.4.3",
-    "prettier": "^1.15.3",
+    "prettier": "1.19.1",
```

Comment:
question: Do configuration defaults really change in minor- or patch-level versions? That would seem like a pretty big breach of semvar.

Reply:
Unclear, but their changelog did indicate they include new options/rename options/deprecate old options in minor versions. 🥲

We were technically probably safe anyway since Prettier is on v2 and I don't think they'll update v1, but I wanted to make sure we reflect best practice anyway. Buried somewhere in their documentation -- sorry for not linking it -- was a recommendation to use exact version.

Reply:
👌